python/cugraph-service/server/cugraph_service_server/cugraph_handler.py

```diff
@@ -52,6 +58,8 @@
     GraphVertexEdgeIDWrapper,
 )
 
+ogb = import_optional("ogb")
```

Comment:
I think you should be able to do this import in the extension instead.

Reply:
importing in the extension causes CGS to hang

Reply:
oh that's strange, we'll have to investigate that separately then.